requirements_bjrnn.txt

```diff
@@ -1,9 +0,0 @@
-matplotlib~=3.3.4
-numpy~=1.19.2
-scipy~=1.5.2
-seaborn=0.11.1
-scikit-learn~=0.24.1
-pandas~=1.2.1
-torchvision=0.2.1=py_2
```

Comment:
I was able to run his benchmark with python 3 and with the version of PyTorch that I have in the requirements now, so I don't think it's an issue.

Reply:
@DrShushen re: BJ-RNN separate PyTorch version

> Are you sure? I was able to run all benchmarks in the paper.

Yes, I needed to set up a separate environment with a downgraded PyTorch. That's why BJ-RNN might have not appeared together with the rest of the benchmarks if you just ran the notebook.

Reply:
Let me check double check this, one moment.

Reply:
Ah, yes, you're right. It seems you don't have that baseline in either notebook at the moment, so I didn't actually run this. I'm happy to not skip checking it at this point. Which version of PyTorch does it need?

Reply:
I'm guessing it would be 1.0.0, as that's compatible with torchvision=0.2.1 (https://pytorch.org/get-started/previous-versions/#v100)

Reply:
Added.

Reply:
Please merge the PR if you're happy with everything else.

Reply:
I think `torchvision=0.2.1=py_2` is what worked for me (hence I put it in the separate requirements file). 

Running the BJ-RNN baseline should be similar to the others, but I only implemented it for synthetic dataset (as in the paper). I lost it somewhere from the notebooks in the process unfortunately, but it _should_ work by running `run_synthetic_experiments` in `train_synthetic.py` with `baseline="BJRNN"` (where it should follow `if baseline == "BJRNN"` block).

(Though a warning it might take a while and take up quite a bit of memory etc...)

Reply:
I'll try to run it with the requirements I added now, will let you know if it works. Let's wait until I can replicate this baseline.

Reply:
Able to run BJ-RNN with this set of requirements:
```
jupyter
matplotlib
numpy
pandas
scikit-learn
scipy
seaborn
torch==1.4.0  # Requires old version of PyTorch.
```

It is quite slow, so I will let it run over the weekend. Will then commit the updated requirements file and a new `synthetic_bjrnn.ipynb` file etc.

Reply:
Thanks so much! (and if unlike me you're able to run it for more than one seed, you're even getting new results... 🙃 )